shared-helpers/src/views/multiselectQuestions.tsx

```diff
@@ -261,7 +261,6 @@ export const getCheckboxOption = (
       {watchFields[optionFieldName] && option.collectAddress && (
         <div className="pb-4">
           <FormAddress
-            subtitle={t("application.preferences.options.address")}
```

Comment:
OK, so this is weird. This subtitle prop was removed in the PR, but recently changed on main. Should we keep it removed, or add it back in??

Reply:
Given the Figma changes I think we're good to keep it removed

Reply:
We're good, this is resolved from the last UIC change